src/app/aliases/aliases.component.html

```diff
@@ -8,23 +8,8 @@
     <mat-progress-bar mode="indeterminate"></mat-progress-bar>
   </div>
   <ng-template #doneLoading>
-    <div *ngIf="organization$ | async as org" fxLayout="column" fxLayoutAlign="center center" fxLayoutGap="10px">
-      <mat-progress-bar mode="indeterminate"></mat-progress-bar>
-      Redirecting to the organization <strong>{{ org.displayName }}</strong>
-    </div>
-    <div *ngIf="collection$ | async as coll" fxLayout="column" fxLayoutAlign="center center" fxLayoutGap="10px">
-      <mat-progress-bar mode="indeterminate"></mat-progress-bar>
-      Redirecting to the collection <strong>{{ coll.displayName }}</strong>
-    </div>
-    <div *ngIf="workflow$ | async as wf" fxLayout="column" fxLayoutAlign="center center" fxLayoutGap="10px">
-      <mat-progress-bar mode="indeterminate"></mat-progress-bar>
-      Redirecting to the workflow <strong>{{ wf.full_workflow_path }}</strong>
-    </div>
-    <div *ngIf="tool$ | async as tool" fxLayout="column" fxLayoutAlign="center center" fxLayoutGap="10px">
-      <mat-progress-bar mode="indeterminate"></mat-progress-bar>
-      Redirecting to the tool <strong>{{ tool.tool_path }}</strong>
-    </div>
-    <mat-card class="alert alert-warning mat-elevation-z" role="alert" *ngIf="types.includes(type) && (aliasNotFound$ | async)">
+    <div *ngIf="validType && found">Redirecting ...</div>
```

Comment:
The above "Redirecting..." have a `mat-progress-bar`. Does this new one need it too?

Reply:
This might be an illusion caused by the diff, the `mat-card` actually belongs to the next block, the original "Redirecting" `divs` aren't in `mat-cards`.  (I could be wrong.)  The original file https://github.com/dockstore/dockstore-ui2/blob/develop/src/app/aliases/aliases.component.html